frontends/web/src/routes/account/manage/manage.tsx

```diff
@@ -0,0 +1,221 @@
+import { getSupportedCoins } from '../../../api/backend';
+import SimpleMarkup from '../../../utils/simplemarkup';
+import { Button, Input } from '../../../components/forms';
+import { Entry } from '../../../components/guide/entry';
+import { Guide } from '../../../components/guide/guide';
+import { Header } from '../../../components/layout';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { Step, Steps } from './components/steps';
+import { CoinDropDown } from './components/coin-dropdown';
+import * as styles from '../manage/manage.css';
+import checkicon from '../../../assets/icons/check.svg';
+
+interface AddAccountProps {
+    // type?: 'multi' | 'btconly'
+}
+
+type Props = AddAccountProps & TranslateProps;
+
+interface State {
+    accountName: string;
+    coinCode: 'choose' | accountApi.CoinCode;
+    onlyOneSupportedCoin: boolean;
+    step: 0 | 1 | 2;
+    supportedCoins: string[];
+}
+
+class ManageAccount extends Component<Props, State> {
+    public readonly state: State = {
+        accountName: '',
+        coinCode: 'choose',
+        onlyOneSupportedCoin: false,
+        step: 0,
+        supportedCoins: [],
+    };
+
+    public componentDidMount() {
+        getSupportedCoins()
+            // TEST with only 1 coin
+            // .then(() => (['btc']))
+            .then((coins) => {
+                const onlyOneSupportedCoin = (coins.length === 1);
+                this.setState({
+                    // @ts-ignore
+                    coinCode: onlyOneSupportedCoin ? coins[0] : 'choose',
+                    onlyOneSupportedCoin,
+                    supportedCoins: coins
+                });
+            })
+            .catch(console.error);
+    }
+
+    private getStep = () => {
+        const { onlyOneSupportedCoin, step } = this.state;
+        if (onlyOneSupportedCoin) {
+            return step === 0 ? 'choose-name' : 'success';
+        }
+        switch (step) {
+            case 0: return 'select-coin';
+            case 1: return 'choose-name';
+            case 2: return 'success';
+        }
+    }
+
+    private back = () => {
+        this.setState(({ step }) => {
+            switch (step) {
+                case 0:
+                case 1: return ({ step: 0 });
+                case 2: return ({ step: 1 });
+            }
+        });
+    }
+
+    private next = () => {
+        const { accountName, coinCode } = this.state;
+        switch (this.getStep()) {
+            case 'select-coin':
+                console.info(`${coinCode} selected`);
+                break;
+            case 'choose-name':
+                // TODO: post accountName and coinCode to backend
+                console.info(`add new account for ${coinCode} with the name: ${accountName}`);
+                break;
+            case 'success':
+                route('/account-summary');
+                return;
+                break;
+        }
+        this.setState((state) => {
```

Comment:
why do you translate between `0 | 1 | 2` and `select-coin | choose-name | success` here and in getStep()? Maybe just use the logical names everywhere?

Reply:
To skip the first step for BB02 Bitcoin-only but still be able to just add + 1 to the step state, but yeah I'll try to remove the step number completelly.

Reply:
Done in https://github.com/digitalbitbox/bitbox-wallet-app/pull/1277